test/e2e/kubernetes/workloads/dns-liveness.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     - while true; do sleep 600; done
     livenessProbe:
       exec:
-        command: ['sh', '-c', 'nslookup www.bing.com || nslookup google.com']
+        command: ['sh', '-c', 'nslookup -type=A www.bing.com || nslookup -type=A google.com']
```

Comment:
Finally making this change so we can test "long probes" directly, and not indirectly by relying upon nslookup's behavior of waiting a few seconds to timeout an AAAA lookup for `www.bing.com`.